Sources/Helpers/Helpers.swift

```diff
@@ -57,3 +57,14 @@ func HWIOtoOWHI(weights: Data, shape: Shape) -> Data {
 
     return Data.init(bytes: transposed, count: shape.totalCount * MemoryLayout<Float>.stride)
 }
+
+public func float32to16(_ input: UnsafeMutablePointer<Float>, count: Int) -> [UInt16] {
+    var output = [UInt16](repeating: 0, count: count)
+    var bufferFloat32 = vImage_Buffer(data: input,   height: 1, width: UInt(count), rowBytes: count * 4)
+    var bufferFloat16 = vImage_Buffer(data: &output, height: 1, width: UInt(count), rowBytes: count * 2)
+
+    if vImageConvert_PlanarFtoPlanar16F(&bufferFloat32, &bufferFloat16, 0) != kvImageNoError {
+        print("Error converting float32 to float16")
```

Comment:
Maybe we need to discuss how to handle errors, but I prefer to fail with a noticeable error here. Maybe an `assert`